### PR TITLE
Add TODO to REAMDE

### PR DESCRIPTION
**Pull request by meiqimichelle**
This adds a very, very basic TODO to the README to help folks know what to expect. Improvements to the words welcome.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -23,3 +23,18 @@
...
 To enable debugging, set `BUILDKITE_ANALYTICS_DEBUG_ENABLED=true`
+
+## Jest Collector Roadmap
+
+### DONE
+
+- [x] Baseline metric collection, such as test duration and pass/fail states
+- [x] Error messaging
+
+### TODO
```
Comment (toolmantim) [inline readme.md]:
```suggestion
## TODO
```

File: readme.md [hunk 2]
```diff
@@ -23,3 +23,18 @@
 5) Run your tests
 
 To enable debugging, set `BUILDKITE_ANALYTICS_DEBUG_ENABLED=true`
+
+## Jest Collector Roadmap
```
Comment (toolmantim) [inline readme.md]: Given this repo is one day going to support other JS test frameworks, perhaps:

```suggestion
## Roadmap
````

File: readme.md [hunk 3]
```diff
@@ -23,3 +23,18 @@
...
+
+### DONE
+
+- [x] Baseline metric collection, such as test duration and pass/fail states
+- [x] Error messaging
```
Comment (toolmantim) [inline readme.md]:
```suggestion
- [x] Send test pass/fail status on test completion
- [x] Send failure reason
```

File: readme.md [hunk 4]
```diff
@@ -23,3 +27,26 @@
...
+#### DONE
+
+- [x] Baseline metric collection, such as test duration and pass/fail states
+- [x] Error messaging
+
+#### TODO
+
+- [ ] Support for deep-dives into spans via records of time spent in HTTP requests or running SQL queries
+- [ ] Support for annotations
+- [ ] Web socket support
```
Comment (toolmantim) [inline readme.md]:
```suggestion
- [ ] Tests
- [ ] Sending through `failure_expanded`
- [ ] HTTP tracing
- [ ] SQL tracing
- [ ] Annotations
- [ ] Live stream results via web sockets
- [ ] Additional test framework support (Cypress , Jasmine, AVA, Mocha, etc)
```

**Review by toolmantim — COMMENTED** [review]
So good!

**Comment by mariovisic** [comment]
Awesome initiative, maybe some folks from the community will want to implement some of these 🙌 

I've had a crack at the HTTP tracing without much luck sadly, would be awesome if someone managed to crack that :)

File: readme.md (hunk 3 above)
Comment (meiqimichelle) [reply readme.md]: I lost your edits when I quickly in-browser resolved some conflicts yesterday @toolmantim . Have a look at my latest commit, should have them back now.

File: readme.md (hunk 4 above)
Comment (meiqimichelle) [reply readme.md]: I lost your edits when I quickly in-browser resolved some conflicts yesterday @toolmantim . Have a look at my latest commit, should have them back now.

File: README.md [hunk 5]
```diff
@@ -1,6 +1,11 @@
 # Buildkite Test Analytics Collector for JavaScript
 
-## Jest
+
+This package collects data about your test suite's performance and reliability, and allows you to see trends and insights about your test suite over time ✨
```
Comment (toolmantim) [inline README.md:4]:
```suggestion
This is the official repo for JavaScript-based [Buildkite Test Analytics](https://buildkite.com/test-analytics) collectors ✨
```

**Review by toolmantim — APPROVED** [review]
This is great! I left one last suggestion. Perhaps we can set the website URL for this repo, in the repo settings, to https://buildkite.com/docs/test-analytics/integrations as well.

**Comment by meiqimichelle** [comment]
@toolmantim should def set the repo URL, good reminder! Tho the structure of our docs is changing as I type -- it'll be safer to set it to https://buildkite.com/docs/test-analytics/ for now. (PS: I don't think I have access to repo settings to add the URL myself)

**Pull request merged**